dockerfiles/Dockerfile.apisix-openresty.rpm

```diff
@@ -16,6 +16,7 @@ ARG IMAGE_TAG
 ENV IMAGE_BASE=${IMAGE_BASE}
 ENV IMAGE_TAG=${IMAGE_TAG}
 
-RUN version=${version} ./build-apisix-openresty-centos7.sh \
+RUN version=${version} \
```

Comment:
Ditto here.

Reply:
Changed to use `ENV` to set environment variables.